src/compiler/scala/tools/nsc/typechecker/TypeDiagnostics.scala

```diff
@@ -10,7 +10,7 @@ import scala.collection.mutable
 import scala.collection.mutable.ListBuffer
 import scala.util.control.Exception.ultimately
 import symtab.Flags._
-import PartialFunction._
+import PartialFunction.{condOpt => whenever}
```

Comment:
Seth's nit: I know, `condOpt` isn't the greatest name, but it's the name we have, I don't think we should indulge ourselves in arbitrary local renamings like this

Reply:
I know what you mean. Paulp once said those methods where the most underlooked of his contributions, and I think it's partly the names. Maybe we can fix that.